test/kubemark/start-kubemark.sh

```diff
@@ -167,6 +139,35 @@ EOF
   echo "Kubeconfig file for kubemark master written to ${LOCAL_KUBECONFIG}."
 }
 
+# Copy all the necessary resource files (scripts/configs/manifests) to the master.
```

Comment:
I don't think it's needed, but it's not bad (the counterargument is that this PR should have 2 lines and has 60)